lang/values.md

```diff
@@ -193,7 +197,7 @@ impl Type {
         assert_eq!(values.len(), fields.len());
         let mut bytes = list![AbstractByte::Uninit; size.bytes()];
         for ((offset, ty), value) in fields.iter().zip(values) {
-            bytes[offset.bytes()..][..ty.size::<M>().bytes()].copy_from_slice(ty.encode::<M>(value));
+            bytes.write_subslice_with_length(offset.bytes(), ty.size::<M>().bytes(), ty.encode::<M>(value));
```

Comment:
`ty.size::<M>()` is redundant here, right? The length of `ty.encode::<M>(value)` already says all we need.